src/app.ts

```diff
@@ -1,22 +1,32 @@
 import { Image } from './image/image.ts';
-
-console.log('hey there pardner');
+import { Toolbar } from './components.ts/toolbar.ts';
```

Comment:
you can drop the .ts from the directory names (`components`) here, and the `mv components.ts components`

Reply:
Good to know, not sure why I that was auto completed for me..

Reply:
Fixed